src/content/docs/en/pages/guides/marketplace/templates/wordpress-edgeaccelerator.mdx

```diff
@@ -70,23 +71,12 @@ Fields identified with an asterisk are mandatory.
 - **Custom host header**: in some cases, it may be necessary to add host information in the header request to retrieve information from the origin. This variable is used for that purpose.
   - For example: `${host}`. Read more about the Host Header and how to customize it on the [Origins documentation](/en/documentation/products/build/edge-application/origins/). 
 
-After completing all the information, click the **Next** button, located in the bottom-right corner. This will start the deployment process.
+After completing all the information, click the **Save** button, located in the bottom-right corfner. This will start the deployment process.
```

Comment:
```suggestion
After completing all the information, click the **Save** button, located in the bottom-right corner. This will start the deployment process.
```